Tests/CryptoTests/CipherTests.swift

```diff
@@ -55,11 +55,28 @@ class CipherTests: XCTestCase {
         let message = "vapor"
         let key = "passwordpasswordpasswordpassword"
         let iv = "123456789012"
-        let (ciphertext, tag) = try AES256GCM.encrypt(message, key: key, iv: iv)
+        var (ciphertext, tag) = try AES256GCM.encrypt(message, key: key, iv: iv)
+        XCTAssertEqual(ciphertext.hexEncodedString(), "4fa166802c")
+
+        // Forcibly overwrite bytes in the tag data to fail authentication
+        tag[2] = 0
```

Comment:
Micro-nit: I'd suggest setting just one byte to zero; that should be sufficient to make the test fail.